src/command/cmd_funcs.c

```diff
@@ -6712,6 +6709,9 @@ cmd_autoaway(ProfWin* window, const char* const command, gchar** args)
     if (g_strcmp0(args[0], "check") == 0) {
         _cmd_set_boolean_preference(args[1], command, "Online check", PREF_AUTOAWAY_CHECK);
         return TRUE;
+    } else {
```

Comment:
I would remove this else block and just add `cons_bad_cmd_usage..` before the `return TRUE` on line 6717.

Reply:
Good idea. Just forced pushed your suggestion.